src/applications/vaos/tests/e2e/vaos-cypress-helpers.js

```diff
@@ -391,16 +389,18 @@ function setupSchedulingMocks({ cernerFacility = false } = {}) {
             ...mockUser.data.attributes.vaProfile,
             facilities: [
               ...mockUser.data.attributes.vaProfile.facilities,
-              {
-                facilityId: cernerFacility,
-                isCerner: true,
-              },
+              { facilityId: cernerFacility, isCerner: true },
             ],
           },
         },
       },
     };
     cy.login(mockCernerUser);
+  } else if (withoutAddress) {
+    const mockUserWithoutAddress = JSON.parse(JSON.stringify(mockUser));
+    mockUserWithoutAddress.data.attributes.vet360ContactInformation.residentialAddress.addressLine1 =
```

Comment:
I think our data utilities work in here, so you could import `unset` from `platform/utilities/data` and do

```
const mockUserWithoutAddress = unset('data.attributes.vet360ContactInformation.residentialAddress.addressLine1', mockUser);
```
without having to clone the mockUser.